### ISSUE TITLE
Custom height thumb does not scroll to bottom of track.

### ISSUE THREAD

**Issue by Chris-DuCharme** [Grsmto/simplebar#455]
**Describe the bug**

**To Reproduce**
1. When the natural height of the scrollbar is taller than you want it and you set a fixed height to make the scrollbar thumb always square.
2. Scroll all the way to the bottom of the div.
3. The scrollbar thumb does not reach the bottom of the track like it does when it's it natural size
4. It appears the space below is equal to the to the difference in height between what you defined and what the natural height would be.

**Expected behavior**
The scrollbar thumb should scroll all the way to the bottom.

**Reproducible example**
Please let me know if you need me to build out an example, the current project I'm working on is covered by an NDA and can't be shared in part.

**Your environment**

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar        | 5.1.0
| Browser          | Chrome
| npm/Yarn         | N/A
| Operating System | MacOS 10.14.6

**Comment by GiorgosK** [comment]
Yes your observation is correct, it happens to me as well, it seems the track height/width does not accommodate for the css changed height of the thumb unfortunately.

**Comment by IrynaVynnychenko** [comment]
so, how can we change height of the scroll, maybe somebody find decision?)

**Comment by junjie-w** [comment]
Yes, I am having the same issue as well - just I need to set the scrollbar thumb longer than the native height, in this case if I scroll to the bottom, the scrollbar thumb will appear shorter than I expected and looks like exactly the height as the native scroll bar thumb.

**Comment by brnohxz** [comment]
Use scrollbarMaxSize={73} and set thumb's height in pixels. In my case I set height to 73px, otherwise I use css styles by provide my custom classes with classNames prop. Hope it will be useful for everyone here!
